portfolio/src/main/webapp/index.html

```diff
@@ -5,14 +5,188 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <nav class="navbar navbar-expand-sm bg-light navbar-light sticky-top" id="nav">
+        <ul class="navbar-nav">
+            <li class="nav-item">
+                <a class="nav-link" style="color:black" href="#gettoknowme">HOME</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" style="color:black" href="#projectshowcase">Project Showcase</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" style="color:black" href="#kodakmoments">Kodak Moments</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" style="color:black" href="#cameraeatsfirst">Camera Eats First</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" style="color:black" href="#step,brush,tap!">Step, Brush, Tap!</a>
+            </li>
+        </ul>
+    </nav>
+    <!--
```

Comment:
It's conventional at Google to never included commented out code in landed PRs.

Reply:
Sorry about that, it was a related to a feature I was working on. Will not happen again!